src/lib/import.js

```diff
@@ -95,6 +95,28 @@ function loadAndValidateConfigFile (fileOrBuffer) {
   return res
 }
 
+/**
+ * Writes default app config to .aio file
+ *
+ * @param {string} parentDir the parent folder to write the .aio file to
+ * @param {object} [flags] flags for file writing
+ * @param {boolean} [flags.overwrite=false] set to true to overwrite the existing .env file
+ * @param {boolean} [flags.merge=false] set to true to merge in the existing .env file (takes precedence over overwrite)
+ * @param {boolean} [flags.interactive=false] set to true to prompt the user for file overwrite
+ * @returns {Promise} promise from writeFile call
+ */
+function writeDefaultAppConfig (parentDir, flags) {
+  // write app config to .aio file
+  const appConfig = {
```

Comment:
as an output, this has to be tested that it doesn't change unexpectedly

Reply:
@shazron there is test for `writeDefaultAppConfig ` in import.test.js to check output

Reply:
Saw the test. I was thinking more like the contents here:
```
app: {
       actions: 'actions',
       dist: 'dist',
       web: 'web-src'
     }
```
A user might be relying on those values we output, so we shouldn't delete or change them (without a major version release) and a test would ensure this contract

Reply:
Added the check to verify default aio content.